native/dlaqr5.go

```diff
@@ -124,9 +124,8 @@ func (impl Implementation) Dlaqr5(wantt, wantz bool, kacc22 int, n, ktop, kbot,
 	}
 
 	// Note: lapack says that nshfts must be even but allows it to be odd
-	// anyway. At the moment, we panic above if nshfts is not even, so
-	// reducing it by one is unnecessary (it seems that Dlaqr0 and Dlaqr4
-	// indeed use only even nshfts).
+	// anyway. We panic above if nshfts is not even, so reducing it by one
```

Comment:
We could do `nshfts := len(sr)`. But but this is a very minor point not worth changing.